requirements.txt

```diff
@@ -4,7 +4,7 @@ PyBrowserID>=0.9.2 # Optional, required for building documentation.
 # Tests
 mock>=0.8.0
 Django>=1.3
-django-nose>=1.2
+nose>=1.3.6
```

Comment:
Does this still use nose as the testrunner? Or are we using nose solely for some of the helper functions? If the latter, you could probably ditch nose, too, and just implement `eq_` and whatever else you're using.